### ISSUE TITLE
Build issue in angular 5 has no exported member 'DSVRowString'.

### ISSUE THREAD

**Issue by anup0651** [d3/d3-dsv#36]
```
ERROR in node_modules/@types/d3-fetch/index.d.ts(9,10): error TS2305: Module '"F:/AprotechSolutions/ems-beam-webapp/node_modules/@types/d3-dsv/index"' has no exported member
'DSVParsedArray'.
node_modules/@types/d3-fetch/index.d.ts(9,26): error TS2305: Module '"node_modules/@types/d3-dsv/index"' has no exported member 'DSVRowString'.
node_modules/@types/d3-fetch/index.d.ts(9,40): error TS2305: Module '"/node_modules/@types/d3-dsv/index"' has no exported member 'DSVRowAny'.
node_modules/@types/d3-fetch/index.d.ts(9,57): error TS2497: Module '"/node_modules/@types/d3-dsv/index"' resolves to a non-module entity
and cannot be imported using this construct.
node_modules/@types/d3/index.d.ts(24,15): error TS2498: Module '"/node_modules/@types/d3-dsv/index"' uses 'export =' and cannot be used with 'export *'.
```

**Comment by roymil** [comment]
Same. Relevant version:
"@types/d3": "5.0.0"
"d3": "5.4.0"
"d3-dsv": "1.0.8"

**Comment by thesourcerer** [comment]
Adding the line:
@types/d3-dsv": "1.0.32",
to package.json
worked around the problem for us.

**Comment by mbostock** [comment]
This seems like a TypeScript or Angular issue, rather than a d3-dsv issue, so closing.

**Comment by thesourcerer** [comment]
I was under the impression that it was caused due to the fact that for @types/d3-dsv, version 0.4.4 was published a bit after 1.0.32, which made things confusing as to which one is the latest,
and this is why giving the "hint" @types/d3-dsv": "1.0.32", in package.json  resolved the issue.
Why are the releases jumping between major 0 and major 1 ? seems a bit odd.
